study.md

```diff
@@ -34,7 +34,17 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+I chose these methods to study because they are used in other languages as well, so I thought it would be important.
+.map
+using map on an array will map a function (that you define within it) and apply it to each element of the array. This will create a new array with these new values. The difference between using an array or an enumerator is the syntax (map{|item|block} -> new array) vs (map{|object| block} -> array)
+
+.collect
+also goes through each element of the array and redefines it depending on what function you put inside.
+
+Looks like the difference between array and enumerable is how you use them. Array takes in an item where enumerable will take in an object. WIth an array, each item will be separated, acted upon, and then brought back to define a new array but for enumerable, the object you define is just placed inside the array in each place.
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.